tests/trainer/test_trainer.py

```diff
@@ -96,42 +101,8 @@ def test_no_val_module(monkeypatch, tmpdir, tmpdir_server, url_ckpt):
         if url_ckpt
         else new_weights_path
     )
-    model_2 = EvalModelTemplate.load_from_checkpoint(checkpoint_path=ckpt_path, hparams_file=hparams_path)
-    model_2.eval()
-
-
-@pytest.mark.parametrize("url_ckpt", [True, False])
-def test_no_val_end_module(monkeypatch, tmpdir, tmpdir_server, url_ckpt):
```

Comment:
Why is this test gone?

Reply:
not sure why this was there. Check the test above. Both are exactly the same.